ci/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apk update && apk add --no-cache curl
 # curl kubectl instead of ADD'ing it, since we want to cache the layer
 RUN curl -sfL \
       -o /usr/local/bin/kubectl \
-      https://storage.googleapis.com/kubernetes-release/release/v1.8.2/bin/linux/amd64/kubectl && \
+      https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && \
```

Comment:
We can make this more generic to pull the latest, see : https://github.com/heptio/kube-conformance/blob/master/Makefile#L22